var/spack/repos/builtin/packages/fast-global-file-status/package.py

```diff
@@ -12,14 +12,22 @@ class FastGlobalFileStatus(AutotoolsPackage):
 
     homepage = "https://github.com/LLNL/FastGlobalFileStatus"
     url = 'https://github.com/LLNL/FastGlobalFileStatus/files/2271592/fastglobalfilestatus-1.1.tar.gz'
+    git = "https://github.com/LLNL/FastGlobalFileStatus.git"
+    maintainers = ['lee218llnl']
 
+    version('master', branch='master')
+    version('1.1.1', sha256='397de583a99e60aae8b4485d3decac6e23f50c658a6353fea149d6dd50d3ecee', url="https://github.com/LLNL/FastGlobalFileStatus/releases/download/v1.1.1/fastglobalfilestatus-1.1.1.tar.gz")
     version('1.1', sha256='e6fba4a0b7f055899fa0e05d93a435c7f1f2ec1158b9a6647dc8d2bcf9c2e164')
 
     depends_on('mrnet')
-    depends_on('mount-point-attributes')
+    # we depend on mpa@master for bug fixes since mpa 1.1
+    depends_on('mount-point-attributes@master')
```

Comment:
@matz-e : I suggested Lee to release a new version. It may not be bad idea to add one version ourselves from master branch commit? As you know, this `@master` might create future headaches.

Reply:
@matz-e : what should we do here? As you know the consequences, I am bit uneasy about deploying `master` in production. We can't undo it. As packages like `FastGlobalFileStatus` are trivial , we can tag some alpha versions ourselves? (By the way, I am bit surprised by accepting `@master` upstream but ...)